server/tests/snapshots/snap_test_multiple_targeted_contests.py

```diff
@@ -48,11 +63,11 @@
 \r
 ######## ROUNDS ########\r
 Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
-1,Contest 1,Targeted,191,Yes,1.84402281147622e-05,DATETIME,DATETIME,candidate 1: 132; candidate 2: 59\r
-1,Contest 2,Targeted,485,No,1.0,DATETIME,DATETIME,Yes: 0; No: 0\r
-1,Contest 3,Opportunistic,,No,1.0,DATETIME,DATETIME,candidate 1: 0; candidate 2: 0; candidate 3: 0\r
-2,Contest 2,Targeted,1468,Yes,8.27580207807509e-24,DATETIME,DATETIME,Yes: 1028; No: 440\r
-2,Contest 3,Opportunistic,,No,1.0,DATETIME,DATETIME,candidate 1: 0; candidate 2: 0; candidate 3: 0\r
+1,Contest 1,Targeted,191,Yes,0.0000184402,DATETIME,DATETIME,candidate 1: 132; candidate 2: 59\r
+1,Contest 2,Targeted,485,No,1.0000000000,DATETIME,DATETIME,Yes: 0; No: 0\r
```

Comment:
this seems a little silly. is there an easy way to say "print as many digits as necessary up to 10" instead of "always print 10 digits"? if not, no biggie

Reply:
This one too, @mcchilders. It should be possible.

Reply:
Since these are percentages represented as decimals, I think preserving all 10 is fine (we won’t ever have 1.0000000000 unless you didn’t actually audit any ballots, so that’s a not-very-likely edge case.)

Reply:
I've fixed it so it strips all trailing zeros except in the `p = 1.0` case.